doc/source/developer_reference/output_processing_api.rst

```diff
@@ -3,14 +3,61 @@
 Output Processing API
 =====================
 
-WA3 now has an output API that can be used to post process a run's
-:ref:`Output Directory Structure<output_directory>` which can be performed by using WA's
-``RunOutput`` object.
-
-Example:
-
-If we have an existing WA output called ``wa_output`` in the current working
-directory we can initialize a ``RunOutput`` as follows:
+WA3 now has an output API that can be used to post process a runs
+:ref:`Output Directory Structure <output_directory>` which can be performed by using WA's
+:class:`RunOutput` object.
+
+This object provides a simple way to access the various aspects of the run. A
```

Comment:
I think this description belongs in `doc/source/developer_reference/APIs.rst` and should just be linked to here.